flake.lock

```diff
@@ -232,9 +294,9 @@
     },
     "hercules-ci-effects": {
       "inputs": {
-        "flake-compat": "flake-compat_4",
+        "flake-compat": "flake-compat_6",
         "hercules-ci-agent": "hercules-ci-agent",
-        "nixpkgs": "nixpkgs_4",
+        "nixpkgs": "nixpkgs_6",
```

Comment:
Why do we have so many nixpkgses. We probably need some follows.

Reply:
You'd have to add follows _recursively_ to every flake input.

Reply:
What do you mean? If N dependencies each use 1 Nixpkgs, we should only need at most N follows to only use 1 Nixpkgs.

Reply:
You haver to do that for dependencies of dependencies too.

```
...
emanote -> tailwind-haskell -> flake-utils -> nixpkgs
emanote -> ema -> flake-utils -> nixpkgs
emanote -> ema -> flake-compat
...
```

All those paths should be set, which I think I did in recent commits.

See https://github.com/srid/emanote/blob/bd2526efc3af2d88590b608c31f89aa115a8840e/flake.nix#L7-L12

Reply:
That is in fact true. This is O(n^2) for n dependencies it seems.